docs/cody/capabilities/supported-models.mdx

```diff
@@ -4,6 +4,8 @@
 
 Cody supports a variety of cutting-edge large language models for use in Chat and Commands, allowing you to select the best model for your use case.
 
+<Callout type="note">Newer versions of Sourcegraph Enterprise can be configured to use newer models as they are introduced. See [Model Configuration](/cody/clients/model-configuration) for more information.</Callout>
+
```

Comment:
```suggestion
<Callout type="note">Newer versions of Sourcegraph Enterprise (starting from [Insert Release Version for Aug 7th] can be configured to use newer models as they are introduced. See [Model Configuration](/cody/clients/model-configuration) for more information.</Callout>
```

Reply:
We need to fill out the release number when we know what it is going to be

Reply:
Is it 5.5.4+?

Reply:
+1 to @aramaraju

Reply:
yeah I'll confirm with the release team, they are only able to bump minor versions every 2 months for now so the aug 7 release is not predictable